### ISSUE TITLE
minecraft 1.21.1 is crashing with the view model versiol 1.21.1

### ISSUE THREAD

**Issue by avionn22** [CyberFlameGO/viewmodel-changer#180]
### Code of Conduct

- [X] I have read and agree to the project's [Code of Conduct](https://github.com/CyberFlameGO/.github/blob/master/docs/CODE_OF_CONDUCT.md)

### What improvements would you like to see?

nothing the mod is good but always crash

### What changes are you suggesting?

no change

### Additional information

no

**Comment by CyberFlameGO** [comment]
Hey - are you able to send a crash report my way? 
Also, are you using Feather? That mod is known to crash when launching alongside this mod

**Comment by avionn22** [comment]
> Hey - are you able to send a crash report my way? Also, are you using Feather? That mod is known to crash when launching alongside this mod

where to find crash report tho , and i think i do have a feather

**Comment by avionn22** [comment]
[crash-2024-11-09_11.49.46-client.txt](https://github.com/user-attachments/files/17687077/crash-2024-11-09_11.49.46-client.txt)

u mean this one?

**Comment by CyberFlameGO** [comment]
Yes, that's perfect thanks! It seems to be an incompatibility with a mod named "smallpop" unfortunately, which is a known issue/side effect of this mod touching the same areas of the client as other mods. If you were to remove "smallpop", the mod should load as expected. I hope this helps :)

**Comment by avionn22** [comment]
> Yes, that's perfect thanks! It seems to be an incompatibility with a mod named "smallpop" unfortunately, which is a known issue/side effect of this mod touching the same areas of the client as other mods. If you were to remove "smallpop", the mod should load as expected. I hope this helps :)

ohhhh Ok i will try to remove smallpop
